CHANGELOG.md

```diff
@@ -1,58 +1,63 @@
 # Changelog
 
-## extendr devel
-
-- Register S3 method `[[.rust_class` just as `$.rust_class`.
-
-- Extra Serialize support.
-
-- Direct Debug support for wrappers.
-
-- Strings::elt returns Rstr.
-
-- Renamed RType to Rtype.
-
-- Removed Bool type.
-
-- Introduced scalars Rint, Rfloat and Rbool.
-
-- Introduced wrappers Integers, Doubles and Logicals.
-
-- Re-written impl Robj as a set of smaller traits.
-
-- Introduced Rany type and as_any.
-
-- Implemented Serialize for R objects.
-
-- Fixed Clippy warnings
-
-- Fixed "myvar" test failures - related to base_env() being the "local" enviroment in the tests.
-
-- Fixes for builds on ARM and PPC platforms.
-
-- Converted R! to a procedural macro, allowing parameters.
-
-- Converted pairlist! to a procedural macro.
-
-- Refactor extendr_macros.
-
-- Remove unused lazy_static
-
-- Continued improvments to wrappers for specific R types such as environments, functions and symbols.
-
-- Install system dependencies on Linux.
-
-- Use Use failure() to trigger steps on failures
-
-- `SymPair::sym_pair()` now returns `(Option<Robj>, Robj)`
-
-- Bump `ndarray` to 0.15.3. Under [RFC 1977](https://github.com/rust-lang/rfcs/pull/1977) this is a "public dependency" change, and therefore can be considered a breaking change, as consumers of `extendr` that use an older version of `ndarray` will no longer be compatible until they also bump `ndarray` to a compatible version.
-
-- Implement `TryFrom<&ArrayBase> for Robj`, allowing `extendr`-annotated functions to return Arrays from the `ndarray` crate and have them automatically converted to R arrays. Note, even if 1D arrays are returned they will not be returned as vectors.
-
-- Removed `TryFrom` conversions between `Robj` and `HashMap` for consistency. `List::into_hashmap()` and `List::from_hashmap()` should be used instead.
-
-- Added support for the graphics-related API (`graphics` feature flag, disabled by default).
+## 0.3.0
+
+### Added
+
+- `Function` type that wraps an R function, which can be invoked using the `call()` method [[#188]](https://github.com/extendr/extendr/pull/188)
+- `pairlist!` macro for generating `Pairlist` objects, e.g. for use in function calls [[#202]](https://github.com/extendr/extendr/pull/202)
+- `use_try_from` option for the `extendr` macro, which allows the use of any type that implements `TryInto<Robj>`/`TryFrom<Robj>`, e.g. `#[extendr(use_try_from = true)]` [[#222]](https://github.com/extendr/extendr/pull/222)
```

Comment:
I though that it should be enabled by default now? I am not even sure `use_try_from` is still used.

Reply:
Hmm, not sure. I believe it's still in the source code. @andy-thomason?

Reply:
Also, it's still an open issue here: https://github.com/extendr/extendr/issues/336

Reply:
I am not sure about the state of this feature. I expect that now it is opt-out (i.e set to `true` by default), but I want to double check before we write it down :)